frontend/ui2/src/demos/layout/MainLayout.tsx

```
@@ -0,0 +1,67 @@
+  activeComponent,
+  onComponentChange
+}: MainLayoutProps) {
+  const navigate = useNavigate();
+  const [isCollapsed, setIsCollapsed] = React.useState(false);
+  const [activeComponentItems, setActiveComponentItems] = React.useState(shadcnComponents);
+
+  // Determine which component list to show based on active component
+
+  const handleItemClick = (id: string) => {
+    if (id === 'shadcn') {
+      debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)